newscoop/src/Newscoop/GimmeBundle/Controller/ArticlesListController.php

```diff
@@ -141,17 +144,13 @@ public function getPlaylistsArticlesAction(Request $request, $id)
         }
 
         $playlistArticles = $em->getRepository('Newscoop\Entity\Playlist')
-            ->articles($playlist, null, false, null, null, $onlyPublished, true)->getResult();
+            ->articles($playlist, array(), true, null, null, $onlyPublished, true)->getResult();
 
-        $articlesIds = array();
+        $articles = array();
         foreach ($playlistArticles as $playlistArticle) {
-            $articlesIds[] = $playlistArticle['articleId'];
+            $articles[] = $playlistArticle->getArticle();
```

Comment:
yeah but here I used `select` from the query builder and not the native query, which returns diffrent structure a bit :)